play-framework/websockets/app/utils/MessageConverter.java

```diff
@@ -0,0 +1,34 @@
+package utils;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import dto.MessageDTO;
+import dto.RequestDTO;
+
+/**
+ * Created by alfred on 07 Jun 2019
+ */
+public class MessageConverter {
+    public static MessageDTO jsonNodeToMessage(JsonNode jsonNode) {
+        ObjectMapper mapper = new ObjectMapper();
+        final MessageDTO messageDTO = mapper.convertValue(jsonNode, MessageDTO.class);
+        return messageDTO;
```

Comment:
no need for the temp variable - just return `mapper.convertValue(...)`'s result

Reply:
Done.